internal/service/rds/instance.go

```diff
@@ -497,6 +497,11 @@ func ResourceInstance() *schema.Resource {
 				Computed:     true,
 				ValidateFunc: validation.StringInSlice(StorageType_Values(), false),
 			},
+			"storage_throughput": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Default:  500,
```

Comment:
This value can only be set when the `storage_type` is `gp3`, so it cannot be the default value

Reply:
No `Default` and use `Computed`.